base/db/R/dbfiles.R

```diff
@@ -350,9 +350,10 @@ dbfile.input.check <- function(siteid, startdate = NULL, enddate = NULL, mimetyp
         container.id = inputs$id,
         con = con,
         hostname = hostname,
-        machine.check = TRUE,
+        machine.check = FALSE, #issues with hostid not matching and resulting in error message dispite files existing on host machine
         return.all = return.all
       )
+      PEcAn.logger::logger.info("Set machine.check = FALSE, when TRUE results in error message even when files are on host machine")
```

Comment:
Should this just be a comment?  Doesn't seem like something the user has control over.  Either that or add it as an argument to the function and give it a default value?

Reply:
It is already a function argument and the default value is TRUE. Recently, I've had issues with the machine check failing even though the files exist on the host machine (which is the point of the check). I added the message to aid in debugging and so that it would clear to others using the function, that for the time being I am changing the default value.

Reply:
I mean that it doesn't look like it's an argument that's being passed from `dbfile.input.check()`.  So the logger message will display when`dbfile.input.check()` is run under certain conditions, but there's no way to change `machine.check` in `dbfile.input.check()`.

Reply:
I switched back to the default values for the PR and am keeping my changes locally.